### PR TITLE
FLA-847 CSO Link

### PR DESCRIPTION
**Pull request by yousiefc**
# Description

This PR includes the following proposed change(s):

- Fla-847
- Added link to CSO account under tab

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

```bash
yarn lint
yarn test -u
yarn test
```

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/efiling-frontend/src/domain/package/package-review/__snapshots__/PackageReview.stories.storyshot [hunk 1]
```diff
@@ -453,6 +453,18 @@ exports[`Storyshots PackageReview Default 1`] = `
         </div>
       </div>
       <br />
+      <b>
```
[the remark this answers is not in the document]
Comment (akroon3r) [reply src/frontend/efiling-frontend/src/domain/package/package-review/__snapshots__/PackageReview.stories.storyshot:456]: the b tag

**Review by akroon3r — COMMENTED** [review]
Just the one comment.

File: src/frontend/efiling-frontend/src/domain/package/package-review/__snapshots__/PackageReview.stories.storyshot (hunk 1 above)
[the remark this answers is not in the document]
Comment (KDNeufeld) [reply src/frontend/efiling-frontend/src/domain/package/package-review/__snapshots__/PackageReview.stories.storyshot:456]: According to the [docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong), both `<b>` and `<strong>` are valid in HTML5, but these aren't exactly the same thing.  Although both are usually rendered as bold face, some browsers will render strong as underlined. 

But in this case, this file is generated test class file and out of our control anyway.

File: src/frontend/efiling-frontend/src/domain/package/package-review/__tests__/PackageReview.test.js [hunk 2]
```diff
@@ -312,4 +313,25 @@ describe("PackageReview Component", () => {
...
+      },
+    });
+
+    const { getByTestId } = render(<PackageReview />);
+    const button = getByTestId("cso-link");
+
+    fireEvent.click(button);
+    await waitFor(() => {});
+
+    // expect(window.open).toHaveBeenCalled()
```
Comment (KDNeufeld) [inline src/frontend/efiling-frontend/src/domain/package/package-review/__tests__/PackageReview.test.js:335]: did you mean to uncomment this assertion?

File: src/frontend/efiling-frontend/src/domain/package/package-review/PackageReview.js [hunk 3]
```diff
@@ -211,6 +220,17 @@ export default function PackageReview() {
             </Tab>
           </Tabs>
           <br />
+          <b>Please Note: </b> Visit your CSO account to{" "}
```
[the remark this answers is not in the document]
Comment (yousiefc) [reply src/frontend/efiling-frontend/src/domain/package/package-review/PackageReview.js:223]: Ya good call, Ill change that

File: src/frontend/efiling-frontend/src/domain/package/package-review/__snapshots__/PackageReview.stories.storyshot (hunk 1 above)
[the remark this answers is not in the document]
Comment (akroon3r) [reply src/frontend/efiling-frontend/src/domain/package/package-review/__snapshots__/PackageReview.stories.storyshot:456]: Thanks for clarification Kevin.

**Review by KDNeufeld — APPROVED** [review]
Looking good!

**Pull request merged**